sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/XmlExternalEntityShouldNotBeParsed_XmlTextReader.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System.Xml;
+
+namespace Tests.Diagnostics
+{
+    class NoncompliantTests_After_Net_4_5_2
+    {
+        // System.Xml.XmlTextReader
+        protected void XmlTextReader_1(XmlUrlResolver parameter)
```

Comment:
I think it would be great to update the method names to reflect what it's being tested. This is valid also for the rest of test classes.